networkit/cpp/distance/test/DynSSSPGTest.cpp

```diff
@@ -442,15 +432,11 @@ TEST_F(DynSSSPGTest, testDynamicDijkstraBatches) {
             dyn_dij.updateBatch(batch);
             DEBUG("Running from scratch with dijkstra");
             dij.run();
-            G.forNodes([&] (node i) {
-            //	std::cout<<"Node "<<i<<":"<<std::endl;
-            //	std::cout<<"Actual distance: "<<dij.distance(i)<<", computed distance: "<<ddij.distance(i)<<std::endl;
-            //	std::cout<<"Actual number of paths: "<<dij.numberOfPaths(i)<<", computed one: "<<ddij.numberOfPaths(i)<<std::endl;
+            G.forNodes([&](node i) {
                 EXPECT_EQ(dyn_dij.distance(i), dij.distance(i));
                 EXPECT_EQ(dyn_dij.numberOfPaths(i), dij.numberOfPaths(i));
                 if (i != source)
-                    assert(dyn_dij.distance(i) != 0);
-            //	EXPECT_EQ(dyn_dij.getPredecessors(i).size(), dij.getPredecessors(i).size());
+                    ASSERT_NEQ(dyn_dij.distance(i), 0);
```

Comment:
Hm, interesting. This should be `ASSERT_NE` - does a wrong linting here.

Reply:
Yes my bad, I'll fix it soon.